Week02/lesson04/homework/relaxr_starter_code/index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<meta name="description" content="Relaxr, the perfect personal assistant. Get Piece of mind with a single tap.">
+	<meta name="keywords" content="Relaxr, Personal Assistant, PA, AI Technology">
+	<title>Relaxr | The Perfect PA</title>
+
+	<!-- external CSS links -->
+	<link rel="stylesheet" href="css/reset.css">
+	<link rel="stylesheet" href="css/style.css">
+	<link href="https://fonts.googleapis.com/css?family=Open+Sans:400,300,300italic,600,800" rel="stylesheet" type="text/css">
+</head>
+
+<body>
+
+	<div id="container">
+		<header>
+			<h1>Relaxr</h1>
+			<h2>Get piece of mind with a single tap</h2>
+			<a href="http://www.relaxr.com/signup"><div id="button1">Get it Now</div></a>
```

Comment:
You seem pretty comfortable turning elements that are `block` by default into `inline-block` in order to make layouts work (like in the Layout Challenge). But as we can see here, sometimes it can seem a little weird to turn an `inline` element into a `block`, so I understand why you've styled the button on the inner `<div>` instead.

However there's no reason to really have that div in this case. You could just have:

`<a id="button1" href="http://www.relaxr.com/signup">Get it Now</a>`

...and then all you would need to add to `#button1` in the CSS is `display: block;` and everything would look just as nice as you have it now.